### PR TITLE
Allow markup in field and non-field error messages

### PR DESCRIPTION
**Pull request by kbayliss**
Addresses https://github.com/torchbox/tbxforms/issues/65

Allows markup to be used within error messages, as well as adding some relevant tests.

### CODE REVIEW

**Review by zerolab — APPROVED** [review]
Yeah, this will do the job. And it is how Django does it (e.g. https://github.com/django/django/blob/86e13843c2ab510fba1de84588efe7fd03555531/django/contrib/admin/templates/admin/login.html#L27-L33)

File: tests/layout/test_errors.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    rendered_form = render_form(form)
+    assert rendered_form == snapshot_html
+    assert "<i>with markup</i>" in rendered_form
+
+
+def test_markup_allowed_in_field_error(form, snapshot_html):
+    """
+    Test markup is rendered for non-field errors.
+    """
+    form.add_error("accept", mark_safe("Field error <i>with markup</i>."))
```
Comment (balazs-endresz) [inline tests/layout/test_errors.py:75]: This is just testing django itself almost but might as well add one more that checks that html is rendered as text when it's NOT marked as safe.

Comment (zerolab) [reply tests/layout/test_errors.py:75]: that is a good shout

Comment (kbayliss) [reply tests/layout/test_errors.py:88]: Good shout - done in https://github.com/torchbox/tbxforms/pull/79/commits/d266f01e0a9f197ff2b801e0cd96d8635767841b

**Pull request merged**